### PR TITLE
Initialize composer

### PR DESCRIPTION
**Pull request by luca-rath**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    "name": "handcraftedinthealps/messenger-utils",
+    "description": "Utilities for symfony messenger.",
+    "type": "library",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "MessengerUtils",
+            "homepage": "https://github.com/handcraftedinthealps/messenger-utils/graphs/contributors"
+        }
+    ]
```
Comment (alexander-schranz) [inline composer.json]: define a php requirement

File: README.md [hunk 2]
```diff
@@ -1 +1,7 @@
-# messenger-utils
\ No newline at end of file
+# MessengerUtils
+
+[![GitHub license](https://img.shields.io/github/license/handcraftedinthealps/messenger-utils.svg)](https://github.com/handcraftedinthealps/messenger-utils/blob/master/LICENSE)
+[![GitHub tag (latest SemVer)](https://img.shields.io/github/tag/handcraftedinthealps/messenger-utils.svg)](https://github.com/handcraftedinthealps/messenger-utils/releases)
+
+Utilities for symfony messenger.
+
```
Comment (alexander-schranz) [inline README.md]: should have only 1 empty line at the end of the file

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# composer
+vendor
+composer.lock
+
```
Comment (alexander-schranz) [inline .gitignore]: should have only 1 empty line at the end of the file

File: README.md (hunk 2 above)
Comment (luca-rath) [reply README.md]: There is only 1 empty line

File: .gitignore (hunk 3 above)
Comment (luca-rath) [reply .gitignore]: There is only 1 empty line

Comment (alexander-schranz) [reply .gitignore]: sorry I mean line breaks. there should not shown an empty line on github then its correct ;)

**Pull request merged**